scripts/samplesheet_generator.py

```diff
@@ -82,7 +83,7 @@ def gen_X_lane_data(pro):
             idxs = find_barcode(sample, pro)
             sp_obj['idx1'] = idxs[0].replace(',','')
             try:
-                sp_obj['idx2'] = idxs[1].replace(',','')
+                sp_obj['idx2'] = str(Seq(idxs[1].replace(',','')).reverse_complement())
```

Comment:
I think that you can probably do this fairly easily in pure python without introducing a new package dependency...

```python
compl = {'A': 'T', 'C': 'G', 'G': 'C', 'T': 'A'} 
sp_obj['idx2'] = ''.join( reversed( [compl.get(b,b) for b in idxs[1].replace(',','') ] ) )
```

(untested)